src/core_modules/capture-ui/DateAndTimeFields/DateField/Date.component.js

```diff
@@ -165,32 +166,37 @@ class UIDate extends React.Component<Props, State> {
                     maxWidth,
                 }}
             >
-                { /* // $FlowSuppress */}
+                { /* // $FlowFixMe */}
```

Comment:
After one of the flow js upgrades, flow has started complaining about "missing" props. These props aren't really missing, but rather passed on through our "passOnProps" object. I don't want to annotate props for every level where they are just passed on, but not used (a "interface" level where every prop is listed could make sense). Suppressing these errors for now, we should upgrade flow at some point and try to find a viable solution to this problem.

Reply:
yeah that makes sense this is a long discussion and you right we need to find a way that flow works in our favour here.